composer.json

```diff
@@ -44,20 +44,21 @@
         "vimeo/psalm": "^4.8"
     },
     "provide": {
-        "container-interop/container-interop-implementation": "^1.2",
```

Comment:
Not sure if we have to keep this for compatibility reasons? 🤷🏼‍♂️

Reply:
In theory not